app/views/request-to-go-live/index.njk

```diff
@@ -91,7 +85,7 @@
               <span class="status">COMPLETED</span>
             {% endif %}
           </h3>
-          <p class="govuk-body-m">GOV.UK Pay will contact you within one working day to confirm whether or not your organisation has already signed an agreement. If not, we’ll send you the relevant documents.</p>
+          <p class="govuk-body-m">To start using GOV.UK Pay your organisation must accept GOV.UK Pay’s legal terms.</p>
```

Comment:
As far I can see `govuk-body-m` isn’t a thing here — https://design-system.service.gov.uk/styles/typography/ and in the code in looks the same as `govuk-body` so maybe best use that instead.